Kitodo-DataManagement/src/main/java/org/kitodo/data/database/beans/Process.java

```diff
@@ -934,6 +930,11 @@ public boolean hasChildren() {
         try {
             return stopwatch.stop(CollectionUtils.isNotEmpty(children));
         } catch (LazyInitializationException e) {
+            if(Objects.isNull(hasChildren)) {
+                this.hasChildren = count(new ProjectDAO(),
+                    "FROM Process AS process WHERE process.parent.id = :project_id",
```

Comment:
This could not be correct: comparing `parent_id` with the id from the  `project`. Did you mean the  `project.id` in process table?

Reply:
it is just the variable name. It works but it is irritating so I change it